### PR TITLE
EDSC-1809: Adds individual order information to Order Status page

### PR DESCRIPTION
**Pull request by trevorlang**

### CODE REVIEW

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 1]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+}
+
+.data-access-collections-list-panel {
+
+  list-style-type: none;
+  margin: 0;
+
+  .data-access-collection {
+    border: 1px solid #bbb;
+    border-left: 5px solid #34495e;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: This same color is used 4 times within this one file. Care to break this out into a variable? #letsgetsassy

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 2]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+.data-access-collection-state {
+  width: 15%;
+  margin-top: 0.16em;
+  padding: .55em;
+  background: #34495e;
+  border-radius: 2px;
+  vertical-align: top;
+  color: #fff;
+  text-align: center;
+  text-transform: capitalize;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: This isn't working in my browser
screen shot 2018-05-08 at 11 10 17 am

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 3]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+  margin-left: 1.5%;
+
+  p {
+    margin: 0;
+  }
 }
 
 .data-access-collection-actions {
-  margin: .5em 0;
+  margin: .75em 0 0;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: Small comment here, and I realize the previous value didn't do this, but let's try to not start values with `.`. Instead, always lead with a 0. So, `0.75em`.

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 4]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+.data-access-services-orders {
+  list-style: none;
+  margin: 1em 0 0;
+  max-height: 385px;
+  overflow: hidden;
+  overflow-y: scroll;
+}
+
+.data-access-services-order {
+
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: Kill empty line here.

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 5]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+  color: #888;
+}
+
+.data-access-order-status {
+  display: inline-block;
+  margin-right: 0.5em;
+  padding: 0.05em .45em;
+  background: #34495e;
   border-radius: 2px;
+  font-size: .85em;
```
Comment (mreese84) [inline app/assets/stylesheets/modules/data_access.css.scss]: Lead this value with a 0.

File: app/assets/stylesheets/modules/data_access.css.scss (hunk 2 above)
Comment (mreese84) [reply app/assets/stylesheets/modules/data_access.css.scss]: Scratch that. I was thinking this was "uppercase". I do prefer `capitalize`.

File: app/assets/javascripts/models/ui/project_list.js.coffee [hunk 6]
```diff
@@ -308,4 +342,19 @@ ns.ProjectList = do (ko
     hideRelatedUrls: ->
       $('#related-urls-modal').modal('hide')
 
+    toggleMoreDetails: (data) =>
```
Comment (abbottry) [inline app/assets/javascripts/models/ui/project_list.js.coffee]: I'd prefer that we use more descriptive variable names here, `data` doesn't tell me what I am being provided.

File: app/assets/javascripts/models/ui/project_list.js.coffee [hunk 7]
```diff
@@ -308,4 +342,19 @@ ns.ProjectList = do (ko
...
+      if data.collection_id not in @moreDetailsActive()
+        @moreDetailsActive.push(data.collection_id)
+      else
+        @moreDetailsActive.splice(@moreDetailsActive().indexOf(data.collection_id), 1)
+
+    toggleViewDownloads: (data) =>
```
Comment (abbottry) [inline app/assets/javascripts/models/ui/project_list.js.coffee]: I'd prefer that we use more descriptive variable names here, `data` doesn't tell me what I am being provided.

File: app/assets/javascripts/models/page/access_page.js.coffee [hunk 8]
```diff
@@ -30,6 +30,7 @@ ns.AccessPage = do (ko,
                     ) ->
 
   class AccessPage
+
```
Comment (abbottry) [inline app/assets/javascripts/models/page/access_page.js.coffee]: Didn't like this being that close? ;)

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 9]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
 
 .data-access-collections-list {
   list-style-type: disc;
-  padding-left: 2em;
+  margin-left: 2em;
+}
+
+.data-access-collections-list-panel {
```
Comment (abbottry) [inline app/assets/stylesheets/modules/data_access.css.scss]: This is a question for @jsiarto and @mreese84 -- do you prefer longer class names or nested class names? These seem like long class names that could just be nested `.data-access-collections` on a surrounding element and `.name` `.body` `.state-text` kind of things nested?

File: app/views/data_access/retrieval.html.erb [hunk 10]
```diff
@@ -24,26 +24,23 @@
...
           <div class="data-access-collection-actions">
             <!-- ko foreach: links -->
-            <a class="button text-button" href="#" target="_blank" data-bind="text: title, attr: {href: url, title: tooltip}" data-tooltip="true" data-placement="bottom"></a>
+            <a class="button button-small text-button" href="#" target="_blank" data-bind="text: title, attr: {href: url, title: tooltip}" data-tooltip="true" data-placement="bottom"></a>
             <!-- /ko -->
           </div>
         </li>
       </ul>
     </section>
     <section class="data-access-content data-access-collection-only" data-bind="visible: ui.projectList.collectionOnly().length > 0">
```
Comment (mreese84) [inline app/views/data_access/retrieval.html.erb]: @trevorlang you can kill this entire section. There is no way to download collection-only data anymore. I just killed this section and ran tests locally, nothing broke. @abbottry maybe you can take a quick look to see if there's anything else associated with this section that can also be removed?

File: app/assets/stylesheets/modules/data_access.css.scss [hunk 11]
```diff
@@ -192,31 +192,151 @@ section.data-access-next {
...
+
```
[the remark this answers is not in the document]
Comment (trevorlang) [reply app/assets/stylesheets/modules/data_access.css.scss]: @mreese84 👍

File: app/assets/stylesheets/modules/data_access.css.scss (hunk 1 above)
Comment (trevorlang) [reply app/assets/stylesheets/modules/data_access.css.scss]: Makes sense. Will do.

File: app/assets/stylesheets/modules/data_access.css.scss (hunk 9 above)
Comment (trevorlang) [reply app/assets/stylesheets/modules/data_access.css.scss]: I do agree that the class name could be shortened possibly. I was mostly trying to follow the naming conventions I saw throughout. I've intentionally avoided nesting to keep specificity low. My BEM habits sneaking in, I suppose. I saw some similar things around the css files. Totally fine with whatever the group prefers though. I'm all ears, @jsiarto and @mreese84.

File: app/views/data_access/retrieval.html.erb (hunk 10 above)
Comment (trevorlang) [reply app/views/data_access/retrieval.html.erb]: 👍Will do

File: app/assets/javascripts/models/page/access_page.js.coffee (hunk 8 above)
Comment (trevorlang) [reply app/assets/javascripts/models/page/access_page.js.coffee]: Sadly, I noticed this one right after I committed. Happy to bring it back to its original home.

File: app/assets/stylesheets/modules/data_access.css.scss (hunk 9 above)
Comment (mreese84) [reply app/assets/stylesheets/modules/data_access.css.scss]: Oddly enough, I don't have a strong opinion here. As long as the class names are semantic/descriptive, I'm good. In projects past we've gotten into some pretty serious specificity wars due to nesting. So while we aren't going full-on BEM here, I do generally support BEM, which would mean more classes and less nesting.

Comment (jsiarto) [reply app/assets/stylesheets/modules/data_access.css.scss]: We should keep nesting to a minimum--it's a nice feature, but can get out of hand quickly. I'm ok with this classname and setup.

**Pull request merged**